projects/worker-animal-pictures/README.md

```diff
@@ -2,7 +2,7 @@
 
 Example of a 3rd party Asset Compute worker. `worker-animal-pictures` produces renditions from an HTTP request to an external API based on an incoming parameter.
 
-It is based on [Project Firefly](https://github.com/AdobeDocs/project-firefly) and the [aio](https://github.com/adobe/aio-cli) developer tool.
+It is based on [App Builder](https://developer.adobe.com/app-builder/) and the [aio](https://github.com/adobe/aio-cli) developer tool.
```

Comment:
`aio` or `App Builder` developer tool?

Reply:
its part the aio cli so I will keep it as the aio developer tool